lib/pavilion/test_config/file_format.py

```diff
@@ -16,10 +16,10 @@ class TestConfigError(ValueError):
 TEST_NAME_RE_STR = r'^[a-zA-Z_][a-zA-Z0-9_-]*$'
 TEST_NAME_RE = re.compile(TEST_NAME_RE_STR)
 KEY_NAME_RE = re.compile(r'^[a-zA-Z][a-zA-Z0-9_-]*$')
+VERS_RANGE_RE = re.compile(r'\d+\.\d+\.\d+-\d+\.\d+\.\d+')
```

Comment:
I don't think you should force a range. Let's accept things like the following:

2.1.0
2.1.*
2.1-2.2.3